libs/sdk-backend-tiger/src/backend/workspace/dashboards/index.ts

```diff
@@ -37,6 +37,7 @@ import {
 } from "../../../convertors/toBackend/AnalyticalDashboardConverter";
 import { TigerAuthenticatedCallGuard } from "../../../types";
 import { objRefToIdentifier } from "../../../utils/api";
+import { ValidateRelationsHeader } from "@gooddata/api-client-tiger/dist/metadataUtilities";
```

Comment:
Please avoid importing from `dist`, `import { ValidateRelationsHeader } from "@gooddata/api-client-tiger";` should work too. This is repeated in all the changed files.